### PR TITLE
Rev version

### PR DESCRIPTION
**Pull request by abliss**
spk released at https://github.com/abliss/gitwebpages-sandstorm/releases/tag/0.0.10

### CODE REVIEW

**Comment by dwrensha** [comment]
Could you please add a `pgp-signature-abliss` file attesting your own authorship and embed it for the `pgpSignature` field, replacing my pgp signature? It was also be good to rename the existing `pgp-signature` file to `pgp-signature-dwrensha`.

Feel free to change the `contactEmail` to point to yourself as well.

**Comment by dwrensha** [comment]
(see https://github.com/sandstorm-io/sandstorm/blob/77a114ca5b401623047ca9a98ccaf9c9cc2920ab/src/sandstorm/package.capnp#L466 )

**Comment by dwrensha** [comment]
And note that Wekan is another app where we handed keys off in this way, when I passed maintainership on to the Wekan Team: https://github.com/wekan/wekan/tree/f275eb5e477f2988289a8544ff7b5171eacb913e/meta

File: .sandstorm/sandstorm-pkgdef.capnp [hunk 1]
```diff
@@ -24,10 +24,10 @@ const pkgdef :Spk.PackageDefinition = (
       categories = [developerTools,],
       author = (
         upstreamAuthor = "Christian Gierke and Kay Sievers",
-        contactEmail = "david@sandstorm.io",
-        pgpSignature = embed "pgp-signature",
+        contactEmail = "abliss@gmail.com",
+        pgpSignature = embed "pgp-signature-abliss",
       ),
-      pgpKeyring = embed "keyring",
+      pgpKeyring = embed "keyring",p
```
Comment (dwrensha) [inline .sandstorm/sandstorm-pkgdef.capnp]: stray `p`?

Comment (abliss) [reply .sandstorm/sandstorm-pkgdef.capnp]: sorry, I had pushed prematurely. now updated with the correct changes, and I put the new spk at https://github.com/abliss/gitwebpages-sandstorm/releases/tag/0.0.10

**Comment by dwrensha** [comment]
Thanks!

**Pull request merged**